include/ace/utils/bitmap.h

```diff
@@ -97,6 +97,22 @@ void bitmapLoadFromFile(
 	tBitMap *pBitMap, char *szFilePath, UWORD uwStartX, UWORD uwStartY
 );
 
+/**
+ *  @brief Loads bitmap data from memory to already existing bitmap.
+ *  If source is smaller than destination, you can use uwStartX & uwStartY
+ *  params to load bitmap on given coords.
+ *
+ *  @param pBitMap    Pointer to destination bitmap
+ *  @param szFilePath Source bitmap pointer.
+ *  @param uwStartX   Start X-coordinate on destination bitmap, 8-pixel aligned.
+ *  @param uwStartY   Start Y-coordinate on destination bitmap
+ *
+ *  @see bitmapCreate
+ *  @see bitmapCreateFromFile
+ */
+void bitmapLoadFromMem( tBitMap *,const UBYTE*, UWORD , UWORD );
```

Comment:
Please add arg names here. It helps in those editors which show function prototype as you type your code - by looking at arg names you don't need to read doxycomment to know what's what.